docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+ postgres:
+  container_name: postgres-case-it
+  image: sdcplatform/ras-rm-docker-postgres
+  ports:
+   - "15432:5432"
+ redis:
+  container_name: redis-case-it
+  image: redis:3.2.9
+  ports:
+   - "17379:6379"
+ rabbitmq:
+  container_name: rabbitmq-case-it
+  image: rabbitmq:3.6.10-management
+  ports:
+    - "34369:4369"
+    - "55672:25672"
+    - "35671:5671"
+    - "35672:5672"
+    - "46671:15671"
+    - "46672:15672"
+ sftp:
```

Comment:
Is the SFTP container needed for this service?

Reply:
No it's not. It has now been removed.